server/pulp/server/db/model/dispatch.py

```diff
@@ -489,3 +490,29 @@ def __init__(self, call_request, call_report):
         self.timestamp = dateutils.now_utc_timestamp()
         self.call_request_string = str(call_request)
         self.serialized_call_report = call_report.serialize()
+
+
+class CeleryBeatLock(Document):
+    """
+    Single document collection which gives information about the current celerybeat lock.
+    """
+
+    """
+        :param celerybeat_name:        string representing the celerybeat instance name
+        :type  celerybeat_name:        basestring
+        :param lock_timestamp:         The timestamp(UTC) at which lock is acquired
+        :type  lock_timestamp:         datetime
+        :param lock:                   A unique key set to "locked" when lock is acquired.
+        :type  lock:                   basestring
+    """
+    celerybeat_name = StringField(required=True)
+    lock_timestamp = DateTimeField(required=True)
```

Comment:
consider changing this field name to `timestamp`. Since the object is named CeleryBeatLock I think it's clear it's a lock_timestamp.